.github/workflows/gotest.yml

```diff
@@ -58,7 +58,7 @@ jobs:
           DATABASE_USER: postgres
           DATABASE_PASSWORD: foobar
           DATABASE_NAME: provisioning_test
-        run: make integration-test
+        run: make integration-test migrations
```

Comment:
```suggestion
        run: make integration-test check-migrations
```

Reply:
Damn, good eyes. I renamed this just as the last thing I thought the prefix is more clear.